ProcessMaker/ImportExport/Exporters/ProcessExporter.php

```diff
@@ -433,4 +427,29 @@ public function importMedia(): void
             $media->model->setAttribute('model_id', $this->model->id);
         }
     }
+
+    /**
+     * Export the process launchpad associated with the process.
+     */
+    public function exportProcessLaunchpad(): void
+    {
+        $launchpad = $this->model->launchpad;
+        if ($launchpad) {
+            $this->addDependent(
+                'process_launchpad',
```

Comment:
Please rename to `launchpad_setting` this is the label requested in the mockup
![image](https://github.com/ProcessMaker/processmaker/assets/42388723/640bb97e-1d85-4298-b9d2-6c2737e5056a)
don't worry about the icon, I was added in my PR